apps/issuer/tests/test_blockchain_service.py

```diff
@@ -0,0 +1,68 @@
+import os
+import binascii
+from issuer.blockchain_service import BlockchainService
+from mainsite import TOP_DIR
+from django.test import TestCase
+from unittest import skip
+
+
+class TestBlockchainService(TestCase):
+
+    def setUp(self):
+        factomd_host = 'http://localhost:8088'
+        factom_walletd_host = 'http://localhost:8089'
+        ec_address = 'EC25UEMqAB6JV61PtfqWymg9ALRPmMZYDep8TiaVfRoRYTeEXwhn'
+        secret_signing_seed = 'fc67c820e0fe764dc9d3536ae753de67cd29ca07a7774849f7e238a471dfcd8a'
+        self.blockchain_service = BlockchainService(factomd_host,
+                                                    factom_walletd_host,
+                                                    ec_address,
+                                                    secret_signing_seed,
+                                                    chain_id='4fe630d0a0ed2ee6cb1c75162cf7a5274339f8db8e1d9f8d6eecaf56e6e537fb')
+
+    @skip('Cannot write/read from the blockchain without factomd running')
```

Comment:
Skip's need to be removed. For now it is okay to have Factomd as a dependency for the tests.